BCA/totalchecking.py

```diff
@@ -0,0 +1,38 @@
+import os
+import glob
+import  pandas as pd
+
+def checkTheSum75():
+    all_files_name=glob.glob("*.json")
+    currentPath=os.getcwd()
+    noFoYears=[]
+    check75=0
+    #loop for opening file 
+    for l in all_files_name:
+        print("checking for Year : ",l)
+        sampleFile=pd.read_json(currentPath+"\\"+l)
```

Comment:
use `os.path.join(currentPath, l)` instead which makes sure that your code runs on both windows and linux and macos